pkg/storage/api_object_versioner.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package etcd
+package storage
```

Comment:
This is an etcd specific package - the interfaces might be storage, but the implementation provided here is not (etcd has int64 resource version, deletiontimestamp, etc)